.github/workflows/daily-test.yml

```diff
@@ -47,6 +50,17 @@ jobs:
             sleep 10
           done
 
+      - name: Run test against the latest 2 iOS versions using no-video-codecs build
+        id: tests-no-video-codecs
+        run: |
+          id=$(curl -F 'payload=@amazon-chime-sdk-app-no-video-codecs.ipa' -F name=amazon-chime-sdk-app-no-video-codecs.ipa -u "${{ secrets.SAUCE_USERNAME }}:${{ secrets.SAUCE_ACCESS_KEY }}" 'https://api.us-west-1.saucelabs.com/v1/storage/upload' |jq '.item.id')
+          npm install
+          npm run build
+          for os in 17 18; do
```

Comment:
I don't think this will work, as we haven't migrated the testing framework. Can you verify?